src/bitshuffle_core.c

```diff
@@ -49,6 +49,8 @@ typedef int64_t omp_size_t;
 typedef size_t omp_size_t;
 #endif
 
+typedef uint16_t alias_uint16_t __attribute__((may_alias));
```

Comment:
Unfortunately this line is incompatible with the Microsoft Visual C compiler. How about:

```C
#if defined(_MSC_VER)
  typedef uint16_t alias_uint16_t;
#else
  typedef uint16_t alias_uint16_t __attribute__((may_alias));
#endif
```

Reply:
Sure. Put in a PR?

Reply:
#177